js/components/utils.js

```diff
@@ -953,6 +953,27 @@ var Utils = {
     }
     return false;
   },
+
+  /**
+   * the function returns the maximum width of the elements
+   * @param {...Element} DOM Element
+   * @returns {number} - max width of the elements
```

Comment:
Can you add in the @private or @public tag? I don't know which it's supposed to be.

Reply:
It was removed